src/details/ArborX_DetailsSortUtils.hpp

```diff
@@ -60,7 +60,7 @@ struct CopyOp<DstViewType, SrcViewType, 1>
 };
 
 template <class DstViewType, class SrcViewType>
-struct CopyOp<DstViewType, SrcViewType, 2>
+struct [[deprecated]] CopyOp<DstViewType, SrcViewType, 2>
```

Comment:
These functors are clearly implementation details as they are defined in the `Details::PermuteHelper` namespace.  Why are you suggesting to deprecate them rather than just removing them?

Reply:
So that we don't use them internally.